Bio/SeqFeature.py

```diff
@@ -851,6 +851,16 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Check if start, end and strand of this location are the same as in the other location"""
+        if not isinstance(other, FeatureLocation):
+            return False
+        return self._start == other.start and self._end == other.end and self._strand == other.strand
```

Comment:
They are strings or None IIRC. Do you fancy adding a test for those too while your looking at this? e.g. in ``Tests/one_of.gb`` we have locations like this:

```
     CDS             join(2201..2479,U18267.1:120..246,U18268.1:130..288,
                     U18270.1:4691..4788,U18269.1:82..>128)
                     /gene="TMPO"
```